### PR TITLE
Delete enhanced file after linting

### PR DESCRIPTION
**Pull request by johanneswuerbach**
Currently the enhanced file is not deleted after it has been linted.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -36,7 +42,7 @@ function evalCodeInMarkdownFile (file, callback) {
   );
 }
 
-function enhanceFile (file) {
+function enhanceFile (file, newPath) {
```
Comment (sdepold) [inline index.js]: I think we are not using the newPath arg anywhere?

File: index.js [hunk 2]
```diff
@@ -36,7 +42,7 @@ function evalCodeInMarkdownFile (file, callback) {
...
   var fileExtension = path.extname(file.path);
   var newFileName   = file.path.replace(/[^\w-]/g, '-') + fileExtension;
   var newPath       = path.join(os.tmpdir(), newFileName);
```
Comment (sdepold) [inline index.js:48]: seems that this var is clashing with the second arg?

File: index.js (hunk 1 above)
Comment (johanneswuerbach) [inline index.js]: Correct, this was a leftover from a previous attempt 😞

**Comment by johanneswuerbach** [comment]
All comments fixed.

**Comment by sdepold** [comment]
Released as 1.1.1

**Comment by sdepold** [comment]
:bow:

**Pull request merged**